qeb_thamos_advise.py

```diff
@@ -122,14 +122,15 @@ def qeb_hwt_thamos_advise() -> None:
     response = advise_using_config(
         pipfile=pipfile_str,
         pipfile_lock=pipfile_lock_str,
-        config=thoth_yaml_config.content,
+        config=json.dumps(thoth_yaml_config.content),
         github_event_type=Configuration._GITHUB_EVENT_TYPE,
         github_check_run_id=Configuration._GITHUB_CHECK_RUN_ID,
         github_installation_id=Configuration._GITHUB_INSTALLATION_ID,
         github_base_repo_url=Configuration._GITHUB_BASE_REPO_URL,
         origin=Configuration._ORIGIN,
         source_type=ThothAdviserIntegrationEnum.GITHUB_APP,
         no_static_analysis=False,
+        src_path=".",
```

Comment:
What if `src_path` is set in `thoth_yaml_config.content`?

Reply:
do we have a key for that? from the logic in thamos it should get the path from `thoth_config.config_path` after giving config content but it returns None after running 

```
    try:
        thoth_config.load_config_from_file(config)
    except (FileNotFoundError, IOError):
        thoth_config.load_config_from_string(config)
```

Reply:
See: https://github.com/thoth-station/thamos/blob/003f6a22c411fba48566f2e9fc5c93dbf2be17be/thamos/lib.py#L342

Reply:
https://github.com/thoth-station/thamos/blob/003f6a22c411fba48566f2e9fc5c93dbf2be17be/thamos/lib.py#L351

Reply:
Yeah, I was wondering if that would always be the behaviour or if it is just the case that you tested that the configuration did not set `src_path`.

Reply:
For now I think we can use this solution and leave that issue to further investigate, wdyt @KPostOffice ?

Reply:
Fine by me I think